documentation/references/apis/01_aleo_public_api.md

```diff
@@ -0,0 +1,71 @@
+---
+id: public_api
+title: Aleo Public API
+sidebar_label: Aleo Public API
+---
+## Overview of Provable's API Offerings
+
+The Provable API is designed to provide users with access to the Aleo blockchain through RESTful endpoints that allow one to query the network for block level information, transactions, transitions, and more. Developers can easily interact directly with Aleo through both retreiving data as well as broadcasting transactions. These provided endpoints are essential for the ecosystem to flourish and grow.
+
+## API Base URLs
+
+<!-- markdown-link-check-disable -->
+| Environment | URL |
+|-------------|-----|
+| Mainnet API | https://api.explorer.provable.com/v1/mainnet |
+| Testnet API | https://api.explorer.provable.com/v1/testnet |
+| Mock Server | https://stoplight.io/mocks/provable1/api-reference/33447 |
```

Comment:
Remove the mock server as this is now within the dev docs and no longer in Stoplight